averbis/core/_rest_client.py

```diff
@@ -938,6 +1064,14 @@ def list_processes(self) -> List[Process]:
             if process.document_source_name == self.name
         ]
 
+    def get_document_identifier(self, document_name: str) -> str:
```

Comment:
The document identifier is going away - we should not have a public method for getting it.